flocker_bb/builders/flocker.py

```diff
@@ -395,6 +396,12 @@ def makeOmnibusFactory(distribution, triggerSchedulers=()):
             "setup.py", "sdist",
             ],
         haltOnFailure=True))
+    if distribution == 'fedora-20':
```

Comment:
One reason to go with the later is (because we always merge forward), forcing a build on a builder that is triggered won't work after somebody merges something new to master. So decreasing those dependencies would be useful. There are probably ways around this, but doing the simpler thing that doesn't require dependencies is probably better in the short term.